src/scripts/components/Play.js

```diff
@@ -12,7 +12,7 @@ import queryString from 'query-string'
 import { PlaybackLevel } from 'records/PlayerRecords'
 import VideoRecord from 'records/VideoRecords'
 import VideoOverlayContainer from 'containers/VideoOverlayContainer'
-import Button from 'components/foundations/Button'
+import Button, { SVGIcon } from 'components/foundations/Button'
```

Comment:
Why is `SVGIcon` part of the `Button.js` file? I think we should separate it. I wouldn't think svg icons need to be coupled to buttons

Reply:
Totally agree, @bent0b0x